packages/react-tabster/src/hooks/useFocusIndicatorStyle.ts

```diff
@@ -1,10 +1,55 @@
 import type { Theme } from '@fluentui/react-theme';
-import type { MakeStylesStyleRule } from '@fluentui/make-styles';
+import type { MakeStyles, MakeStylesStyleRule } from '@fluentui/make-styles';
 import { KEYBOARD_NAV_SELECTOR } from '../symbols';
 
-const defaultStyleRule = (theme: Theme) => ({
-  outline: `solid 1px ${theme.alias.color.neutral.neutralForeground1}`,
-});
+export type FocusOutlineOffset = Record<'top' | 'bottom' | 'left' | 'right', string>;
+export type FocusOutlineStyleOptions = {
+  /**
+   * Only property not supported by the native CSS `outline`, if this is no longer needed
+   * we can just go native instead
+   */
+  outlineRadius: string;
+  outlineColor: string;
+  outlineWidth: string;
+  outlineOffset?: string | FocusOutlineOffset;
+};
+
+/**
+ * NOTE: the element with the focus outline needs to have `position: relative` so that the
+ * pseudo element can be properly positioned.
```

Comment:
Couldn't you add `position: relative` as part of the styles you return in `createFocusOutlineStyle`?

Reply:
Potentially, but I'm not entirely sure about changing the `position` css property in a pseudo-selector... it could lead to UI jumpiness ? but I'm not 100% certain on this